alerts/machine-config-operator/MachineConfigControllerDrainError.md

```diff
@@ -0,0 +1,114 @@
+# MCCDrainError
+
+## Meaning 
+
+Alerts the user to when the Machine Config Operator 
+failed to drain a node. Preventing the MCO from restarting 
```

Comment:
Somehow, putting this in present tense seems better to me. A purely optional suggestion.
```suggestion
fails to drain a node, which prevents the MCO from restarting 
```

Reply:
Agreed, Applied the change